common/system/filtering/filters.cpp

```diff
@@ -127,7 +146,8 @@ static double filterAccelOneAxis(double* accelReadings, double* imuReadings,
   }
 #endif  // HAS_DEV(IMU)
 
-  return numAccelsValid == 0 ? 0.0 : accelSum / numAccelsValid;
+  return numAccelsValid == 0 ? imuReadings[highestFsIndex]
```

Comment:
This will only use the imu with the highest full scale rather than choosing the best from the imu and accel sensors (accels generally being the high-g accelerometer). We'd like it to be able to use either, particularly because the high g accelerometers we probably will want to fall back on won't be "imus" as far as the code goes.

Our naming is sort of confusing so let me know if that doesn't make sense.

Reply:
Lol ok yeah I thought these were like mutually exclusive things